patterns/hidden-comments.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
I wish twe didn't have to store templates in the patterns folder in order for them to be translatable. I guess there's no real way around that?

Reply:
No. I don't know if there is an open issue that covers it better than this one:
https://github.com/WordPress/gutenberg/issues/20966

The project is closer to solving now that there are block bindings.
_In theory,_ block bindings could already be used, since you can use custom PHP functions which could include the translations.... but I don't think this theme should experiment with it.

Reply:
Also, this particular part of the theme is not a template or a part, it is intended to be a pattern so that it can be inserted anywhere, and it would be PHP file even if it had no translatable strings.